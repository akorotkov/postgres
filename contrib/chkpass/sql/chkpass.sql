CREATE EXTENSION chkpass;

CREATE TABLE test (i int, p chkpass);
INSERT INTO test VALUES (1, 'hello'), (2, 'goodbye');

SELECT i, p = 'hello' AS "hello?" FROM test;
SELECT i, p <> 'hello' AS "!hello?" FROM test;
