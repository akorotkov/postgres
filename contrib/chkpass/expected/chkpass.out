CREATE EXTENSION chkpass;
WARNING:  type input function chkpass_in should not be volatile
CREATE TABLE test (i int, p chkpass);
INSERT INTO test VALUES (1, 'hello'), (2, 'goodbye');
SELECT i, p = 'hello' AS "hello?" FROM test;
 i | hello? 
---+--------
 1 | t
 2 | f
(2 rows)

SELECT i, p <> 'hello' AS "!hello?" FROM test;
 i | !hello? 
---+---------
 1 | f
 2 | t
(2 rows)

